.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ on:
     branches:
       - '**'
 env:
-  NODE_VERSION: 14.18.1
+  NODE_VERSION: 16.10.0
```

Comment:
Can you please use `16.13.0` which is the latest?

Reply:
I thought 16.10.0 was intended, because of the `vm.Script` memory leaks encountered with Jest on Node `16.11.0+` (see https://github.com/facebook/jest/issues/11956)

Anyway, you're using Jasmine, I think it should be OK, let's try.